src/zampy/datasets/utils.py

```diff
@@ -54,3 +68,137 @@ def get_file_size(fpath: Path) -> int:
         return 0
     else:
         return fpath.stat().st_size
+
+
+def cds_request(  # noqa: PLR0913
```

Comment:
Perhaps we should just silence PLR0913 altogether :shrug:

Reply:
Yes, couldn't agree more. It is very annoying (well given that we know what we are doing, those many arguments are added for a reason.). I will silent it completely in `project.toml`.